templates/batch-llm/README.ipynb

```diff
@@ -274,28 +228,89 @@
     "ds = ds.map_batches(\n",
     "    LLMPredictor,\n",
     "    # Set the concurrency to the number of LLM instances.\n",
-    "    concurrency=num_llm_instances,\n",
+    "    concurrency=4,\n",
     "    # Specify the number of GPUs required per LLM instance.\n",
-    "    num_gpus=num_gpus_per_instance,\n",
+    "    num_gpus=1,\n",
```

Comment:
When I ran this, I got

"""
    raise ValueError(
ValueError: Bfloat16 is only supported on GPUs with compute capability of at least 8.0. Your Tesla T4 GPU has compute capability 7.5. You can use float16 instead by explicitly setting the`dtype` flag in CLI, for example: --dtype=half.
"""

Similar to in https://github.com/anyscale/templates/pull/148 I think you need to set `accelerator_type: A10G` and/or make a function that returns A10G or L4 depending on AWS or GCP.

Reply:
ah my bad, i was testing this on a custom workspace with A10s already configured, that makes sense. Adding a similar function as [#148](https://github.com/anyscale/templates/pull/148) which gets A10G/L4 depending on the cloud platform.